parsers/ENTSOE.py

```diff
@@ -628,16 +628,17 @@ def validate_production(datapoint, logger):
 
 
 def get_biomass(values):
-    if 'Biomass' in values or 'Fossil Peat' in values or 'Waste' in values:
-        return values.get('Biomass', 0) + \
-               values.get('Fossil Peat', 0) + \
-               values.get('Waste', 0)
+    if 'Biomass' in values or 'Waste' in values:
+        return (values.get('Biomass', 0)
+                + values.get('Waste', 0))
 
 
 def get_coal(values):
-    if 'Fossil Brown coal/Lignite' in values or 'Fossil Hard coal' in values:
-        return values.get('Fossil Brown coal/Lignite', 0) + \
-               values.get('Fossil Hard coal', 0)
+    if True and ('Fossil Brown coal/Lignite' in values or 'Fossil Peat' in values
```

Comment:
True?

Reply:
debatable, but I don't like this indentation
```
    if ('Fossil Brown coal/Lignite' in values or 'Fossil Peat' in values
        or 'Fossil Hard coal' in values):
        return (values.get('Fossil Brown coal/Lignite', 0)
```
but I agree that adding `True` is confusing